utils/swift-xcodegen/README.md

```diff
@@ -0,0 +1,108 @@
+
+## Command usage
+
+```
+USAGE: swift-xcodegen [<options>] <build-dir>
+
+ARGUMENTS:
+  <build-dir>             The path to the Ninja build directory to generate for
+
+LLVM PROJECTS:
+  --clang/--no-clang      Generate an xcodeproj for Clang (default: --no-clang)
+  --clang-tools-extra/--no-clang-tools-extra
+                          When generating a project for Clang, whether to include clang-tools-extra (default: --clang-tools-extra)
+  --lldb/--no-lldb        Generate an xcodeproj for LLDB (default: --no-lldb)
+  --llvm/--no-llvm        Generate an xcodeproj for LLVM (default: --no-llvm)
+
+SWIFT TARGETS:
+  --swift-targets/--no-swift-targets
+                          Generate targets for Swift files, e.g ASTGen, SwiftCompilerSources. Note
+                          this by default excludes the standard library, see '--stdlib-swift'. (default: --swift-targets)
+  --swift-dependencies/--no-swift-dependencies
+                          When generating Swift targets, add dependencies (e.g swift-syntax) to the
+                          generated project. This makes build times slower, but improves syntax
+                          highlighting for targets that depend on them. (default: --swift-dependencies)
+
+RUNNABLE TARGETS:
+  --runnable-build-dir <runnable-build-dir>
+                          If specified, runnable targets will use this build directory. Useful for
+                          configurations where a separate debug build directory is used.
+  --runnable-targets/--no-runnable-targets
+                          Whether to add runnable targets for e.g swift-frontend. This is useful
+                          for debugging in Xcode. (default: --runnable-targets)
+  --build-runnable-targets/--no-build-runnable-targets
+                          If runnable targets are enabled, whether to add a build action for them.
+                          If false, they will be added as freestanding schemes. (default: --build-runnable-targets)
+
+PROJECT CONFIGURATION:
+  --compiler-libs/--no-compiler-libs
+                          Generate targets for compiler libraries (default: --compiler-libs)
+  --compiler-tools/--no-compiler-tools
+                          Generate targets for compiler tools (default: --compiler-tools)
+  --docs/--no-docs        Add doc groups to the generated projects (default: --docs)
+  --stdlib, --stdlib-cxx/--no-stdlib, --no-stdlib-cxx
+                          Generate a target for C/C++ files in the standard library (default: --stdlib)
+  --stdlib-swift/--no-stdlib-swift
+                          Generate targets for Swift files in the standard library. This requires
+                          using Xcode with with a main development snapshot (and as such is disabled
```

Comment:
nit:
```suggestion
                          using Xcode with a main development snapshot (and as such is disabled
```

Reply:
Resolving in https://github.com/swiftlang/swift/pull/77415